### DISCUSSION TITLE
Using Vue-virtual-scroll-grid with axios

### DISCUSSION THREAD

**Discussion by Aditi-1400** [rocwang/vue-virtual-scroll-grid#69] [Q&A] [answered]
Hey everyone, I've been wanting to use this npm package for one of my projects and it looks great to me.
However, I've been trying and I am unable to use axios within the callback function. Can anyone please help me with it? :)

**Comment by rocwang** [comment, marked as answer]
Basically, you need to write a function that accepts a page number and a page size, then uses axios to fetch the items from your backend in a paginated way. Pass this function to the vue-virtual-scroll-grid component as the `pageProvider` prop. [See the example here.](https://codesandbox.io/s/vue-virtual-scroll-grid-esm-with-axios-n3yu2?file=/App.vue)

P.S. If you have further question, please post it in [Discussions](https://github.com/rocwang/vue-virtual-scroll-grid/discussions) instead. Issues is not for general "how to" help.

**Reply by bot08** [reply]
example is not available :(

**Reply by rocwang** [reply]
> example is not available :(

Sorry for the inconvenience. [The example](https://codesandbox.io/s/vue-virtual-scroll-grid-esm-with-axios-n3yu2?file=/App.vue) is restored now.
